NAMESPACE

```diff
@@ -3,3 +3,7 @@ import(RJSONIO)
 importFrom(utils, packageDescription)
 
 exportPattern("^[[:alpha:]]+")
+
+S3method("pbGetDevices", "default")
```

Comment:
AFAICT that is not needed as the wildcard `exportPattern` takes care of this.  If I do `ls(pos=p)` for the given `p`, I see the symbols already.